eoscostarica/eos-rate
# starts adapting the frontend to a container-only deployment.

refs #https://github.com/eoscostarica/eos-rate/issues/366.

### GH Issue

# Created a frontend container for eos-rate based on the current CI one

This is required to be able to deploy all the services, including the frontend, in a k8s cluster.

### Steps to test

1. run make start
1. visit localhost:3000
1. you should see the eos rate page

### Checklist
